tests/nnstreamer_filter_tensorflow/runTest.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+##
+## @file runTest.sh
+## @author HyoungJoo Ahn <hello.ahn@samsung.com>
+## @date Dec 17 2018
+## @brief SSAT Test Cases for NNStreamer
+##
+TF_OK=0
+PB_OK=0
+
+OUTPUT="$(find /usr/lib -name tensorflow.pc 2>&1 | grep -v 'Permission denied')"
+LENGTH=${#OUTPUT}
+if (( $LENGTH <= 0 ))
+then
+    OUTPUT="$(find /usr/local/lib -name tensorflow.pc 2>&1 | grep -v 'Permission denied')"
+    LENGTH=${#OUTPUT}
```

Comment:
As @wooksong has recommended yesterday,
why don't you try to link the tensorflow-filter library with ```ldconfig -l```?

Reply:
using `ldconfig -l` result is applied :)